src/components/TextOnlyCard/TextOnlyCardWithPrice.jsx

```diff
@@ -1,57 +1,75 @@
-import React from 'react'
-import { BiMinus,BiRupee } from "react-icons/bi";
+import React from "react";
+import { BiMinus, BiRupee } from "react-icons/bi";
 
+export const TextOnlyCardWithPrice = ({ cartState }) => {
+  const { cartItem } = cartState;
+  const DeliveryCharges = 50;
+
+  // calculating total original price
+  const totalOriginalPrice = [...cartItem].reduce(
+    (acc, currentItem) => currentItem.qty * currentItem.originalPrice + acc,
+    0
+  );
+  //  calculating total sellingPrice
+  const totalSellingPrice = [...cartItem].reduce(
+    (acc, currentItem) => currentItem.qty * currentItem.sellingPrice + acc,
+    0
+  );
+  // calculating Bag discount (original-selling)
```

Comment:
You don't need to destructure `cartItem` here, `reduce does not mutate original Array object

Reply:
okay